class.jetpack.php

```diff
@@ -4776,11 +4776,25 @@ public static function delete_secrets( $action, $user_id ) {
 	 * @since 2.6
 	 * @return int
 	 **/
-	public function get_remote_query_timeout_limit() {
```

Comment:
Should we find a way to deprecate `get_remote_query_timeout_limit()` to minimize errors if other plugins are using this?